Pod/Classes/Terminals/Utility/GlobalMobileSDK/GMSRequestHelper.swift

```diff
@@ -63,7 +63,7 @@ class GMSRequestHelper {
                                         cardData: cardData!,
                                         requestMultiUseToken: requestMultiUseToken ?? false)
         } else {
-            return AuthTransaction.auth(total: decimalToUint(total),
+            return AuthTransaction.auth(clientTransactionId: "", total: decimalToUint(total),
```

Comment:
The value for `clientTransactionId` should be obtained from the builder when present.